package.json

```diff
@@ -25,7 +25,9 @@
     "check-format": "prettier --check \"{src,tests}/**/**\"",
     "format": "prettier --write \"{src,tests}/**/**\"",
     "check": "prettier --check \"{src,tests}/**/**\"",
```

Comment:
Not related to this PR, but this seems a duplicate of line 25.

Reply:
True, will remove one.